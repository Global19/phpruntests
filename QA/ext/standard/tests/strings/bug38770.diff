003+     [1] => -30000
003-     [1] => 4294937296