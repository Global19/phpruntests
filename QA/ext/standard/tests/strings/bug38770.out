Array
(
    [1] => -30000
)
Array
(
    [1] => -30000
)
Array
(
    [1] => -30000
)
Done